ntbs-service/Pages/Notifications/Edit/PreviousHistory.cshtml

```diff
@@ -11,14 +11,21 @@
 
 <partial name="_SinglePageErrorSummaryPartial"/>
 
-<h2>Previous History for New Patient</h2>
+<h2>Previous TB History </h2>
 
 <div>
     <div asp-validation-summary="ModelOnly" class="text-danger"></div>
 
     <nhs-form-group nhs-form-group-type="Standard" id="PatientTbHistory-PreviousTBDiagnosisYear">
         <nhs-fieldset>
-            <nhs-fieldset-legend nhs-legend-size="Standard"> The patient has had TB in the past </nhs-fieldset-legend>
+            <nhs-fieldset-legend nhs-legend-size="Standard"> 
+                Please record any previous occurrence of TB for this patient which has not been notified to PHE through 
+                ETS, LTBR or NTBS (for example: because the case was notified outside England, Wales or Northern Ireland, 
+                or before the year 2000).  
+            </nhs-fieldset-legend>
+            <nhs-hint nhs-hint-type="Standard" id="id-types-hint"> 
+                
+            </nhs-hint>

```

Comment:
Did using the hint to work well in the end then? If not, we should remove this bit!

Reply:
If it did, then I think you want to a) put text in it and b) use an `aira-described-by` thingy on the fieldset, see html of the first example on https://design-system.service.gov.uk/components/radios/ (sadly the nhs page doesn't seem to care :( )